external/apb/data-sync.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0; URL=''" />
```

Comment:
data sync is a new service so this doesn't need a link right now. it can stay here as a placeholder